autoload/sy/repo.vim

```diff
@@ -116,6 +116,10 @@ function! s:handle_diff(options, exitval) abort
     let sy.detecting -= 1
   endif
 
+  if has('iconv')
```

Comment:
I many cases `&enc` and `&fenc` will be the same, most probably `utf-8` nowadays. Maybe we should add `&& &fenc != &enc` here, otherwise we'll have a lot of useless `iconv()` calls.

Reply:
Thanks for reviewing. I added a commit.